.travis.yml

```diff
@@ -37,6 +40,11 @@ matrix:
         - RUNJSHINT=true
     - python: 3.6
       env: TOXENV=pii-annotations
+  allow_failures:
```

Comment:
These seem to be passing now, can this section be removed?  It's a problem if tests fail in the configuration that we're currently using in production.

Reply:
Sure, I'll remove this section.